packages/elements/src/number-field/__test__/number-field.validity.test.js

```diff
@@ -24,3 +24,13 @@ describe('number-field/Validity', () => {
     });
   });
 });
+
+describe('Check Floating point', function () {
+  // Test Floating point precision issue that results approximation of real number. e.g. 1111111/0.00001 should equal to 111111100000. 
+  it('Input remains valid upon value update with a step of float value', async function () {
+    const el = await fixture('<ef-number-field step="0.00001" value="111111"></ef-number-field>');
```

Comment:
Can we remove this 
```suggestion
    const el = await fixture('<ef-number-field step="0.00001"></ef-number-field>');
```
and add a comment that value change is needed in order to trigger validation. Also can we have at least one more test case with different number and decimal points ?

Reply:
Added more test cases.